### DISCUSSION TITLE
Python 3.7 on mingw

### DISCUSSION THREAD

**Discussion by advoskina-sc** [msys2/MINGW-packages#17481] [Q&A] [answered]
Hi!
I'm porting old 3.7 python project to windows 10. Need to use mingw to be able to install dependencies (PyGobject).
Is there any way to install python 3.7 on msys2? In repo min available version for downgrading is 3.9. 
thank you!

**Comment by Biswa96** [comment, marked as answer]
> Is there any way to install python 3.7 on msys2?

No. You have to compile that old version manually. I think it would be easier to port the python program to the latest available python version in mingw packages.
